modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/publisher/test/APIApplicationPublisherTest.java

```diff
@@ -234,7 +251,8 @@ public void removedService(
 		}
 	}
 
-	private APIApplication _createAPIApplication(String baseURL, String title)
+	private APIApplication _createAPIApplication(
+			String baseURL, String apiApplicationERC, String title)
```

Comment:
the parameter `apiApplicationERC` should be before the `baseURL` alphabetically sorted. Maybe better if you rename it to `String externalReferenceCode`